packages/ui/core-components/src/lib/atoms/inputs/dropdown/Virtual.svelte

```diff
@@ -135,11 +139,12 @@
 	});
 </script>
 
+<!-- kwongz added tenerary operation for scroll depending on items and displayed options value -->
 <div
 	bind:this={viewport}
 	bind:offsetHeight={viewport_height}
 	on:scroll={handle_scroll}
-	style="height: {height};"
+	style="height: {height}; overflow: {items < displayedOptions ? 'hidden' : 'auto'};"
```

Comment:
We should double check that we need this here - modifying the height (currently 32px) per option to being slightly higher is probably the fix we want here